melos.yaml

```diff
@@ -3,12 +3,12 @@ repository: https://github.com/woltapp/wolt_modal_sheet
 
 packages:
   - . ## This is the root package
-  - coffee_maker
-  - coffee_maker_navigator_2
-  - demo_ui_components
-  - playground
-  - playground_navigator2
-  - wolt_di
+  # - coffee_maker
+  # - coffee_maker_navigator_2
+  # - demo_ui_components
+  # - playground
+  # - playground_navigator2
+  # - wolt_di
```

Comment:
Commented these "test" folders in these folders don't exist for some reason as a result `melos test` return non-zero exit code.